gradlew.bat

```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
Can you remove this? I don't support Windows and if someone wants to try it there, they can manually create the file.